lib/utils/matrix_sdk_extensions/event_extension.dart

```diff
@@ -12,25 +11,25 @@ import 'package:fluffychat/utils/string_extension.dart';
 import 'package:fluffychat/utils/twake_snackbar.dart';
 import 'package:flutter/foundation.dart';
 import 'package:flutter/material.dart';
-import 'package:future_loading_dialog/future_loading_dialog.dart';
 import 'package:flutter_gen/gen_l10n/l10n.dart';
 
 import 'package:matrix/matrix.dart';
 
 import 'matrix_file_extension.dart';
 
 extension LocalizedBody on Event {
-  Future<LoadingDialogResult<MatrixFile?>> getFile(
-    BuildContext context,
-  ) async =>
-      TwakeDialog.showFutureLoadingDialogFullScreen(
-        future: downloadAndDecryptAttachment,
+  Future<void> saveFile(BuildContext context) async {
+    TwakeSnackBar.show(context, L10n.of(context)!.downloadStarted);
+    try {
+      final matrixFile = await downloadAndDecryptAttachment();
```

Comment:
When I tried to test more carefully with bigger file:
- Your modification doesn't block user in loading dialog
- But, it still blocks user to interact with UI. This piece of code make the app freezes.

Reply:
fixed